scheduler.c

```diff
@@ -109,80 +109,74 @@ rb_scheduler_timeout(struct timeval *timeout)
 }
 
 VALUE
-rb_scheduler_kernel_sleep(VALUE scheduler, VALUE timeout)
+rb_fiber_scheduler_kernel_sleep(VALUE scheduler, VALUE timeout)
 {
     return rb_funcall(scheduler, id_kernel_sleep, 1, timeout);
 }
 
 VALUE
-rb_scheduler_kernel_sleepv(VALUE scheduler, int argc, VALUE * argv)
+rb_fiber_scheduler_kernel_sleepv(VALUE scheduler, int argc, VALUE * argv)
```

Comment:
Kernel#sleep always take a single argument, is there any reason to expose this variadic variant, in addition to the one above?